source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/OnlineMediaLoader.java

```diff
@@ -0,0 +1,121 @@
+package io.adaptivecards.renderer;
+
+import android.annotation.TargetApi;
+import android.media.MediaDataSource;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+import android.util.Log;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by almedina on 7/30/2018.
+ */
+
+@TargetApi(Build.VERSION_CODES.M)
+public class OnlineMediaLoader extends IOnlineMediaLoader
+{
```

Comment:
This should be in the test app for people to understand how to implement it and for us to validate, but should not be in our package.